src/docs/patterns/internationalization.mdx

```diff
@@ -13,42 +16,83 @@ import Image4 from '../../app/assets/design-patterns/internationalization/web-in
+While this usually goes with the locale settings, certain users might have different preferences, and there can be variations even within the same locale. You should provide the user with default settings when they first load a locale, but allow the users change their preferences, and remember it the next time they login with that locale.
+
+## Checking for Spacing
+
+Different languages/writing systems may require different vertical and horizontal space. To ensure flexibility, avoid using fixed dimensions for necessary UI components such as buttons, labels, and badges. Also, consider setting different default display font sizes for languages that may have smaller glyphs.
+
+> **NOTE:** PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/#sans-lgc) for all non-western languages. You can read more on our [Typography](/style/typography) page.
+
+## Bi-directionality/Right-to-Left (RTL) Support
 
 <MaterialDesignDescription
     title={'Bidirectionality'}
     description={'Learn about mirroring layout, mirroring elements, and localization.'}
     url={'https://material.io/design/usability/bidirectionality.html'}
 />
 
-## Technical Implementation
+Some languages, such as Arabic and Hebrew, are read from right to left. Applications supporting these languages will need to mirror the UI so the natural flow is from right to left. For example, navigation drawers should be displayed on the right side of the screen, icons should be to the right of the text in buttons, and any icons with directionality should be mirrored.
+
+<ImageGrid
+    images={[UIFlippedImage]}
+    regularWidth
+    caption={[
+        '1: A PX Blue user interface in right-to-left language (English).',
+        '2: The same PX Blue user interface in left-to-right language (Arabic).',
+    ]}
+/>
+<ImageGrid
+    images={[IconsFlippedImage]}
+    regularWidth
+    caption={[
+        '1: Icons that indicate directions should be mirrored.',
+        '2: Icons with glyphs and symbols should never be mirrored.',
+    ]}
+/>
+
+## Curtural differences
+
+Sometimes certain UI elements convey different meaning in different cultures. For example, in America, the red color in the stock market indicates a decline, while in China the same color indicates a rise. A thumb-up gesture usually expresses an approval in English-speaking country, yet the gesture can have a negative meaning in Iraq. When designing for a new market, you are encouraged to have a local consultant run an audit for your application.
+
+<Divider />
+
+# Developers
 
 There are a number of available libraries to assist with implementing internationalization in your applications.
 
-### Angular
+## Angular
```

Comment:
and Ionic?

Reply:
I wasn’t sure what library to use for Ionic. Should I just change the headline to “Angular and Ionic”?

Reply:
I'm assuming it would use the same as angular since it's just a wrapper....similar to using all the same components

Reply:
We haven't actually done an RTL/I18N example in Ionic yet though